extensions/functions_arithmetic.yaml

```diff
@@ -226,9 +243,9 @@ scalar_functions:
           rounding:
             values: [ TIE_TO_EVEN, TIE_AWAY_FROM_ZERO, TRUNCATE, CEILING, FLOOR ]
           on_domain_error:
-            values: [ NAN, ERROR ]
+            values: [ NAN, "NULL", ERROR ]
           on_division_by_zero:
-            values: [ LIMIT, NAN, ERROR ]
```

Comment:
Is this going to be a broken change because we no longer support LIMIT?

Reply:
Technically, yes.  However, I do not know of any engine that implements LIMIT.  So, practically speaking, no.

Reply:
cudf

```
import cudf
a = cudf.Series(5)
b = cudf.Series(0)
a.divide(b)
0    inf
dtype: float64
```

Reply:
Well, darn.  I'll add limit back in then.  I wouldn't introduce limit for this case (since cudf is the only engine) but I think we can keep an option even if there aren't two engines.

Reply:
I've added LIMIT back as an option.